shared/annotation.py

```diff
@@ -642,12 +643,30 @@ def detect_and_remove_collisions(self, instance_list):
                 # Collision detected, we keep the newest instance by created time (which was order sorted).
                 # So this one is just to be deleted and not added to results.
                 logger.warning('Collision detected on {} instance id: {}'.format(inst.hash, inst.id))
+                logger.warning('hashes_dict: {}'.format(hashes_dict))
                 inst.soft_delete = True
                 inst.action_type = "from_collision"
                 self.session.add(inst)
 
         return result
 
+    def rehash_existing_instances(self, instance_list):
+        result = []
+        for instance in instance_list:
+            prev_hash = instance.hash
+            instance.hash_instance()
+            new_hash = instance.hash
+            if prev_hash != new_hash:
+                logger.warning('Warning: Hashing algorithm upgrade Instance ID: {} has changed \n from: {} \n to: {}'.format(
```

Comment:
Super small suggestion - I'm ok either way
I noticed we get some support tickets where people question the memory `warning` thing. If the default use case here is system upgrades, should we do `info` maybe?

Reply:
Sure Anthony! That's a good idea I guess is less invasive for end users that way :)